lms/templates/student_account/institution_register.underscore

```diff
@@ -0,0 +1,32 @@
+<div class="other-login-wrapper">
```

Comment:
When adding structural or styling-minded wrapper elements, we add the ``wrapper-`` prefix. Please change to ``wrapper-other-login``.